airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -35,6 +36,9 @@
 from airflow.utils.log.json_formatter import JSONFormatter
 from airflow.utils.log.logging_mixin import LoggingMixin
 
+# Elasticsearch hosted log type
+EsLogMsgType = List[Tuple[str, str]]
```

Comment:
Are those related changes?

Reply:
Since the execution of one task instance will be processed in two steps, the logs generated from these two steps might come from different host. In es_task_handler, the logs are fetched by offset and it could mess up logs from two hosts. The change in here is to change the returning logs from string to a list of hosted log string so that users can have a readable log url.